buildkitd/Earthfile

```diff
@@ -58,8 +58,9 @@ buildkitd:
     ENTRYPOINT ["/usr/bin/entrypoint.sh", "buildkitd", "--config=/etc/buildkitd.toml"]
     ARG EARTHLY_TARGET_TAG_DOCKER
     ARG TAG="dev-$EARTHLY_TARGET_TAG_DOCKER"
-    ARG DOCKERHUB_USER="earthly"
-    SAVE IMAGE --push --cache-from=earthly/buildkitd:main $DOCKERHUB_USER/buildkitd:$TAG
+    ARG REGISTRY_USER="earthly"
+    ARG IMAGE_REGISTRY="docker.io"
```

Comment:
I still think a comment here describing the difference between podman and docker would be helpful.